src/main/java/ch/puzzle/eft/controller/SiteController.java

```diff
@@ -23,7 +29,9 @@ public SiteController(ExamService examFileService) {
     }
 
     @GetMapping("/")
-    public String viewIndexPage(Model model) {
+    public String viewIndexPage(@CookieValue(value = "cookie-consent", defaultValue = "not-set") String cookieConsent, Model model) {
+        boolean cookiesMissing = !"true".equals(cookieConsent);
```

Comment:
simplify by parsing to boolean

Reply:
Switched the true to the end of equals so its easy readable